examples/complete/variables.tf

```diff
@@ -7,13 +7,11 @@ variable "ibmcloud_api_key" {
 variable "region" {
   type        = string
   description = "Region to provision all resources created by this example"
-  default     = "us-south"
 }
 
 variable "prefix" {
   type        = string
   description = "Prefix to append to all resources created by this example"
-  default     = "kms-key"
```

Comment:
Where did these changes come from?  In all other kms (all-inclusive and rings) the examples have default values for region and prefix; so that any consumer does not need to set them as variables. The tests override them, but with default values it means the consumer (which includes me when running a plan against the source) need to set three variables rather than one.

Reply:
@shemau we are removing default values as per internal issue https://github.ibm.com/GoldenEye/issues/issues/5505